### PR TITLE
Close #27 native dev on win10 documentation

### LINKED ISSUE #27: As a Developer I want to know how to install native development environment on Windows 10

### PR DESCRIPTION
**Pull request by dimus**

### CODE REVIEW

**Comment by LocoDelAssembly** [comment]
@dimus (also applying to the docker flavor) how can you make this env access files on Windows-side? For instance, downloading a project dump from a sandbox and then trying to run `bundle exec rake tw:db:load file=[dump-path]"`?

**Comment by LocoDelAssembly** [comment]
Guess this would be it: https://docs.microsoft.com/en-us/windows/wsl/faq#how-do-i-access-my-c-drive

File: development/native/win_10.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+```bash
+git clone https://github.com/sstephenson/rbenv.git ~/.rbenv
+git clone https://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+git clone https://github.com/rkh/rbenv-update.git ~/.rbenv/plugins/rbenv-update
+```
+
+to `~/.bashrc` add line:
+
+```bash
+eval "$(rbenv init -)"
```
Comment (LocoDelAssembly) [inline development/native/win_10.md:176]: @dimus when doing this and restarting the terminal first line reads `Command 'rbenv' not found`. Per https://github.com/rbenv/rbenv#basic-github-checkout looks we need `echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc` here.

Comment (dimus) [reply development/native/win_10.md:176]: good catch

**Comment by dimus** [comment]
> @dimus (also applying to the docker flavor) how can you make this env access files on Windows-side? For instance, downloading a project dump from a sandbox and then trying to run `bundle exec rake tw:db:load file=[dump-path]"`?

windows files should be straightforward via '/mnt/c/....'

windows can also access linux files. I guess it is done via ssh fuse or something and goes like '//wsl$/home/...'

**Comment by dimus** [comment]
I squashed it into one commit, if all is OK, I can move it to master now

**Pull request merged**